src/modules/tasks.js

```diff
@@ -1,49 +1,53 @@
-/* eslint-disable  import/no-cycle */
 /* eslint-disable import/extensions */
```

Comment:
Please, don't disable linter checks. Linter checks are placed in order to improve the code quality of students. Kindly remove the linter disable rules. I would recommend fixing the linter errors instead of disabling the linter checks.

Reply:
Hello @muneebulrehman thank you for the review, but in my case particularly if you check the import is done properly but eslint still highlights it as an error. I have to add the import/extension disable for the import to work.

Also if I do this
`import { updateTaskStatus } from './modules/taskStatus';`
I be able to remove the disable and I will not have the error but my file will not be found and my code will not work.
As for the no-loop func my bad, I was supposed to remove that. 
I hope you can understand and /or propose a solution for me if you see the import is done wrongly.